### PR TITLE
show changes to controller actions and views

### PR DESCRIPTION
**Pull request by beccaelenzil**
I added the code changes to ada books needed to point to the database and not the hardcoded data in the BOOKS hash for reference

### CODE REVIEW

**Comment by beccaelenzil** [comment]
[rails textbook curriculum - added code for changes in controller, views, and tests for pointing to the database in lesson](https://trello.com/c/wA4uJPuX/2111-rails-textbook-curriculum-added-code-for-changes-in-controller-views-and-tests-for-pointing-to-the-database-in-lesson)

File: 08-rails/active-record-models.md [hunk 1]
```diff
@@ -232,6 +232,64 @@ In the rails console:
 1. Create another book
     - What is the new book's ID? Does ActiveRecord reuse IDs?
 
+## Changes to Ada Books app - Live Code
```
Comment (CheezItMan) [inline 08-rails/active-record-models.md:235]: Maybe just change the title.

```suggestion
## Adjust the controller
```

File: 08-rails/active-record-models.md [hunk 2]
```diff
@@ -232,6 +232,64 @@ In the rails console:
...
+```
+#show.html.erb
+<h1><%= book.title %></h1>
+<h2>By: <%= @book.author %></h2>
+```
+
+Update show test to create a book and use its id
+
+```
+describe "show" do
```
Comment (CheezItMan) [inline 08-rails/active-record-models.md:273]: Just suggest you put the filename/path in here.
```suggestion
# test/controllers/books_controller_test.rb
describe "show" do
```

File: 08-rails/active-record-models.md [hunk 3]
```diff
@@ -232,6 +232,64 @@ In the rails console:
 1. Create another book
     - What is the new book's ID? Does ActiveRecord reuse IDs?
 
+## Changes to Ada Books app - Live Code
+
+First remove hard coded BOOKS hash and then replace references to the `BOOKS` hash in the controller actions with references to the database.
+
+```
+#books_controller.rb
```
Comment (CheezItMan) [inline 08-rails/active-record-models.md:240]:
```suggestion
#  app/controllers/books_controller.rb
```

**Comment by beccaelenzil** [comment]
This material is in the lesson "Active Record in Code" -- I added the lesson to the read me for rails.

**Pull request closed without merging**